Submitted By: Ryan Oliver <ryan(dot)oliver(at)pha(dot)com(dot)au>
Date: 2005-10-20
Initial Package Version: 5.8.7
        Rediffed against 5.10.0 by Joe Ciccone on 2008-08-30
Origin: Ryan Oliver
Description: this patch allows perl to be installed in /usr/lib32 or /usr/lib64 instead of /usr/lib.

diff -Naur perl-5.10.0.orig/Configure perl-5.10.0/Configure
--- perl-5.10.0.orig/Configure	2007-12-18 05:47:07.000000000 -0500
+++ perl-5.10.0/Configure	2008-08-31 11:56:58.000000000 -0400
@@ -6174,6 +6174,8 @@
 : The default "style" setting is made in installstyle.U
 case "$installstyle" in
 *lib/perl5*) set dflt privlib lib/$package/$version ;;
+*lib32/perl5*) set dflt privlib lib32/$package/$version ;;
+*lib64/perl5*) set dflt privlib lib64/$package/$version ;;
 *)	 set dflt privlib lib/$version ;;
 esac
 eval $prefixit
@@ -6406,6 +6408,8 @@
 case "$sitelib" in
 '') case "$installstyle" in
 	*lib/perl5*) dflt=$siteprefix/lib/$package/site_$prog/$version ;;
+	*lib32/perl5*) dflt=$siteprefix/lib32/$package/site_$prog/$version ;;
+	*lib64/perl5*) dflt=$siteprefix/lib64/$package/site_$prog/$version ;;
 	*)	 dflt=$siteprefix/lib/site_$prog/$version ;;
 	esac
 	;;
@@ -6912,6 +6916,8 @@
 		prog=`echo $package | $sed 's/-*[0-9.]*$//'`
 		case "$installstyle" in
 		*lib/perl5*) dflt=$vendorprefix/lib/$package/vendor_$prog/$version ;;
+		*lib32/perl5*) dflt=$vendorprefix/lib32/$package/vendor_$prog/$version ;;
+		*lib64/perl5*) dflt=$vendorprefix/lib64/$package/vendor_$prog/$version ;;
 		*)	     dflt=$vendorprefix/lib/vendor_$prog/$version ;;
 		esac
 		;;
